Tests/Common/Securities/PatternDayTradingMarginBuyingPowerModelTests.cs

```diff
@@ -86,9 +86,9 @@ public void SetLeverageTest()
         public void VerifyOpenMarketLeverage()
         {
             // Market is Open on Tuesday, Feb, 16th 2016 at Noon
-
+            // SPY @ $100 * 100 Shares / Leverage (4) = 2500
             var leverage = 4m;
-            var expected = 100 * 100m / leverage + 1;
```

Comment:
These `+1`'s were from the fee ($1) that got changed to 0. 

This was intentionally changed to zero because other tests in the same set, asserted expected results that did not include calculations for a fee. Specifically `VerifyMarginCallOrderLongClosedMarket`, `VerifyMarginCallOrderShortOpenMarket`, `VerifyMarginCallOrderShortClosedMarket`, & `VerifyMarginCallOrderShortOpenMarket`.

To match their expected calculation I removed the fee from the security all these test share. 👍🏽